example-playbooks/replace_node/replace_node.yml

```diff
@@ -256,6 +271,13 @@
         state: absent
       when: _replace_node_first_boot_grep.failed
 
+    - name: Remove the ignore_dead_nodes_for_replace record
+      lineinfile:
+        path: /etc/scylla/scylla.yaml
+        regexp: '^ignore_dead_nodes_for_replace:'
+        state: absent
+      when: dead_nodes_to_ignore|length > 0
```

Comment:
What if we are using a `scylla` version that expects IPs/hostnames in `ignore_dead_nodes_for_replace`? For example, 2022.2.19?

Those would be same versions that use `replace_address_first_boot` and not the new `replace_node_first_boot`

Reply:
I've adjusted the PR to also support scylla versions that still use addresses instead of host ids in `ignore_dead_nodes_for_replace`